test/integration/tests/psm.ts

```diff
@@ -366,23 +373,113 @@ describe('e2e-peg-stability-module', function () {
         expect(psmEndingDAIBalance.sub(psmStartingDAIBalance)).to.be.equal(mintAmount / 2);
       });
 
-      it('mint succeeds with 10_000_000 DAI', async () => {
-        const minAmountOut = await daiPSM.getMintAmountOut(mintAmount);
+      it('DAI price sanity check', async () => {
+        const actualDAIAmountOut = await daiPSM.getMintAmountOut(mintAmount);
+        await expectApprox(actualDAIAmountOut, mintAmount);
+      });
+    });
+  });
+
+  describe('rai_psm', async () => {
+    describe('redeem', function () {
+      const redeemAmount = 10_000_000;
+      beforeEach(async () => {
+        await fei.connect(impersonatedSigners[minterAddress]).mint(userAddress, redeemAmount);
```

Comment:
recommend adding a check to see if the rai psm needs more rai, and if so dripping or minting rai into it so the test is less brittle when things change on mainnet.

Reply:
good call, will add

Reply:
breaking out to separate issue